.semgrep/let.yml

```diff
@@ -0,0 +1,6 @@
+rules:
+  - id: let-unit-in
+    pattern: let () = $E in ...
+    message: use ; instead
```

Comment:
Does naivley following this advice sometimes lead to bugs? Thinking of

~~~Ocaml
if true then
   let () = do_something () in
   more_stuff()
~~~


vs

~~~OCaml
if true then
    do_something();
    more_stuff()
~~~


Maybe add a caution here ("carefully checking if parens are needed").